src/res/failcode.en.json

```diff
@@ -615,5 +615,8 @@
     "stopHalfway.roma": "A cell cannot reach a goal.",
     "tapaloopError.tapaloop": "The segments around a clue are not the same length as the numbers.",
     "tentAround.tents": "Some tents touch.",
-    "unusedCell.pencils": "A cell is unused."
+    "unusedCell.pencils": "A cell is unused.",
+    "segUnclued.railpool": "The segment's length isn't in the region's set of numbers",
+    "clueUnused.railpool": "Some of the region's numbers don't match with any segment's length",
+    "incompatibleSets.railpool": "The set of lengths of the region's segments doesn't match the region's set of numbers"
```

Comment:
This particular error message is pretty confusing. Is it about the amount of distinct lengths in a region that contains question marks?

Reply:
Yep - this error happens when no there are no valid values for the question marks. For example, a "2 ? ?" clue can never fit in a region with segments of lengths 2 & 3. I've tried several wordings but nothing convinces me - please suggest some alternate error message.

Reply:
At Menderbug's suggestion, I've separated this error into two versions: too many & too few questions marks. The wording in these is much clearer.